src/core/init/initialize_media_source.ts

```diff
@@ -225,52 +230,74 @@ export default function InitializeOnMediaSource(
    * Wait for the MediaKeys to have been created before opening the MediaSource,
    * after that second step is done, ask the EMEManager to attach the MediaKeys.
    * Steps are done in that specific order to avoid compatibility issues.
+   *
+   * This Observable will emit when ready both the MediaSource and useful
+   * DRM-specific information.
    */
   const prepareMediaSource$ = emeManager$.pipe(
-    mergeMap((evt) => {
+    mergeScan((
```

Comment:
😮

Reply:
Is that even what I want to do? I have to re-check. Yet another arguments where rxjs is too complex :p

Reply:
It is!